sympy/matrices/repmatrix.py

```diff
@@ -46,7 +46,7 @@ class RepMatrix(MatrixBase):
     #
 
     _rep: DomainMatrix
-
+#ping
     def __eq__(self, other):
         # Skip sympify for mutable matrices...
         if not isinstance(other, RepMatrix):
```

Comment:
Only do `_sympify` for `MatrixBase` and mpmath matrices. This could possibly be extended to also compare with lists or numpy arrays.